keyboards/mothwing/rev1/rules.mk

```diff
@@ -0,0 +1,20 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = caterina
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = no       # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = no       # Mouse keys
+EXTRAKEY_ENABLE = no       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no        # Commands for debug and configuration
+NKRO_ENABLE = no            # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no      # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no       # Enable keyboard RGB underglow. 
+AUDIO_ENABLE = no           # Audio output
+
+DEFAULT_FOLDER = mothwing/rev1
```

Comment:
```suggestion
```

Reply:
if I completely remove this file, the `qmk compile` command would not find the keyboard right?  
your suggestion is like to delete all the lines(which should be in the `info.json`)? is that correct?  
thank you.

Reply:
Yes, the rules.mk file still needs to exist, even if it's empty. You might also consider not haivng a `rev1/` folder at all unless you are sure you will have more revisions in future (and even then, the folder structure can be updated later).

Reply:
> You might also consider not haivng a rev1/ folder at all unless you are sure you will have more revisions in future (and even then, the folder structure can be updated later).  
I see. thanks!